presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaConnectorModule.java

```diff
@@ -50,6 +51,7 @@ public void configure(Binder binder)
         binder.bind(KafkaConnector.class).in(Scopes.SINGLETON);
 
         configBinder(binder).bindConfig(KafkaConfig.class);
+        newSetBinder(binder, TableDescriptionSupplier.class).addBinding().toProvider(KafkaTableDescriptionSupplier.class).in(Scopes.SINGLETON);
```

Comment:
do we really need more than one description supplier at the same time?

Reply:
we use two in tests